services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -268,12 +278,16 @@ def get_dynamic_sidecar_spec(
                 "CapabilityAdd": [
                     "CAP_LINUX_IMMUTABLE",
                 ],
-                "Labels": {
-                    # NOTE: these labels get on the tasks and that is also useful to trace
-                    "user_id": f"{scheduler_data.user_id}",
-                    "study_id": f"{scheduler_data.project_id}",
-                    "uuid": f"{scheduler_data.node_uuid}",
-                },
+                "Labels": StandardSimcoreDockerLabels(
+                    user_id=scheduler_data.user_id,
+                    project_id=scheduler_data.project_id,
+                    node_id=scheduler_data.node_uuid,
+                    product_name=scheduler_data.product_name,
+                    simcore_user_agent=scheduler_data.request_simcore_user_agent,
+                    swarm_stack_name=dynamic_sidecar_settings.SWARM_STACK_NAME,
+                    memory_limit=ByteSize(0),  # this should get overwritten
```

Comment:
memory and cpu limits are set here to zero which is what you called UNDEFINED above, right? 
So this undefined fits well to set "limits" values but not somehting else

Reply:
well I did not change the logic in here right. the point here is to standardize the labels and make them docker compatible.